app/controllers/turbo/streams/redirect.rb

```diff
@@ -0,0 +1,27 @@
+module Turbo::Streams::Redirect
+  extend ActiveSupport::Concern
+
+  def redirect_to(options = {}, response_options = {})
+    turbo_frame = response_options.delete(:turbo_frame)
+    location = url_for(options)
+
+    respond_to do |format|
```

Comment:
Using `respond_to` here causes an `ActionController::RespondToMismatchError` to be raised when redirecting inside another `respond_to` block:

```
respond_to do |format|
  format.html { redirect_to articles_url }
end
```

Reply:
Thank you for raising this concern @martinzamuner! I've pushed up a change to remove the `respond_to` from the implementation in favor of checking the request format directly.

Could you try to reproduce your issue with the newer version?